096-drop-travis-ci-and-testing-for-ppc-and-s390x.md

```diff
@@ -0,0 +1,51 @@
+# Drop Travis-CI and testing efforts for `ppc64le` and `s390x` architectures
+
+This proposal suggests to drop current usage of [Travis CI](https://www.travis-ci.com/) in [strimzi-kafka-bridge](https://github.com/strimzi/strimzi-kafka-bridge) and [strimzi-kafka-oauth](https://github.com/strimzi/strimzi-kafka-oauth) repositories.
+
+## Motivation
+
+In the past, we adopted Azure Pipelines as the primary CI system for the Operators repository, using it to run tests, build documentation, and create artifacts and images.
+The main reason for this transition was the improved resource quotas available for CNCF projects, which allowed us to run pipelines with far fewer restrictions than we had in Travis CI. 
+However, Azure Pipelines does not provide agents for the `ppc64le` and `s390x` architectures.
+This was not an issue for Operators, as we build multi-architecture images on `amd64` using _docker buildx_. 
+However, for Strimzi Kafka Bridge and Strimzi Kafka OAuth, we decided to retain the testing pipelines for `ppc64le` and `s390x`.
+
+In recent months, we have encountered several unexpected issues with Travis CI that required us to submit support tickets to restore our quota.
+These issues are often not immediately noticed when they occur but are only discovered weeks later. 
+It seems that no one even realizes that some pipelines fail to execute from time to time.
+Another scenario is that jobs fail due to various Travis CI issues, yet we simply ignore the results and merge the PRs anyway.
+
+Such cases create an unexpected maintenance burden with little added value:
+- we must keep Travis CI configurations up to date, even though it is not our primary CI system and is only used for unit tests on the `ppc64le` and `s390x` architectures
+- we occasionally need to contact Travis CI support to resolve issues
+- no one actively reports bugs in `ppc64le` and `s390x` that could have been detected by pipelines running on Travis CI
+- we generally do not pay attention to unstable pipelines on Travis CI and merge PRs regardless
+
+_Note that Strimzi's Travis CI quota is sponsored by IBM._
+
+## Proposal
+
+This proposal suggests removing the existing Travis CI configurations from the Strimzi Kafka Bridge and Strimzi Kafka OAuth repositories, effectively eliminating Travis CI usage from the Strimzi organization.
+Regarding the current support matrix, we will not make any changes, meaning that all currently supported architectures — `amd64`, `arm64`, `ppc64le`, and `s390x` — will continue to be supported by the Strimzi project.
+
+We will newly use the following terminology:
+- the `amd64` architecture will have the status `supported`, which means that all available tests will be run on this architecture on regular basis as we do now
+- the `ppc64le`, `s390x`, and `arm64` architectures will have the status `supported, not tested`, meaning that tests will not be regularly executed on these architectures
```

Comment:
`arm64` can be tested using TF in the operators repo. Or you meant it will be `supported, not tested` because we are not using TF in the operators repo regularly + there is no TF in these two repositories?

Reply:
Yes, we do not run the tests regularly and we do not use it for running unit and IT tests. I decided to make it more restrictive in the proposal than it is in reality, but it is still true - builds and the main part of testing is done only for `amd64`

> there is no TF in these two repositories
I think the app is installed, but yes, there is no config right now. I mentioned also som reasons in `Rejected Alternatives`

Reply:
Okay, thanks.